### DISCUSSION TITLE
Add a plus button at the bottom to create a new tab

### DISCUSSION THREAD

**Discussion by zeroarst** [Able-Bear-Software/tab-nodes-tree#34] [Ideas / Feature Request]
A quick-add button at the bottom for users to easily create a new tab could be helpful. Adding settings like holding Shift while clicking to open it as a child tab might be useful too.

**Comment by rocketeeer** [comment]
Double clicking in the empty space to create a new tab would also be nice

**Reply by zeroarst** [reply]
Thank you for the suggestion, @rocketeeer!

While I can see how double-clicking in the empty space to create a new tab might be convenient for some users, I currently want to keep the focus on more universally accessible options like buttons and key combinations, which are easier to discover for a wider range of users.

That said, I’ll keep your idea in mind for future updates, especially if more users express interest. I appreciate the feedback and am always open to considering different approaches!

Thanks again!

**Comment by cschanck** [comment]
Alternatively, a '+' at the top (Next to the "Tab Nodes Tree" header would work (and also in each group header as well). But having to go to the upper right of the screen to add a new tab is a pain when you use the panel on the  left.

**Reply by zeroarst** [reply]
Hi @cschanck 
Unfortunately developers have no control of the header. I hope one day Chrome team can introudce new API for it.
Do you know you can right click to add new tabs? Or by using shortcut Ctrl+T in Windows. Not sure about macOS.
I might add a setting for user to decide where they want to place the "+" button, whether at the top, bottom or both.

**Comment by cschanck** [comment]
Well, there must be a way because other extensions do it; SideTab Pro, for
example (
https://chromewebstore.google.com/detail/sidetab-pro-vertical-tabs/fehfojhhnbfbclgpffmffigfgngbpnmj?hl=en&pli=1)
does both the collapsing and a new tab in this group button in the header.
So it is possible.

C

Christopher 'BC' Schanck

***@***.***
***@***.***

**Reply by zeroarst** [reply]
Oh sorry. I meant "Tab Nodes Tree" header. For group headers, definitely can be implemented.

**Comment by zeroarst** [comment]
Created two tickets.
https://github.com/Able-Bear-Software/tab-nodes-tree/issues/35
https://github.com/Able-Bear-Software/tab-nodes-tree/issues/63

**Comment by cschanck** [comment]
Excellent! Really nice piece of software.

Christopher 'BC' Schanck

***@***.***
***@***.***

**Comment by Tab-Nodes-Tree** [comment]
Released in [1.5.0](https://github.com/Able-Bear-Software/tab-nodes-tree/discussions/84).
